.circleci/config.yml

```diff
@@ -161,3 +316,5 @@ workflows:
         - equal: ["generate-release", << pipeline.schedule.name >>]
     jobs:
       - generate-release
+          context:
+            - slack-secrets
```

Comment:
We're using the same context as audius-protocol which defaults the channel to #eng-deploy-announcements using the CircleCI user I created.

Reply:
Not sure why this line is causing build errors when it's similar to audius-protocol:

* https://github.com/AudiusProject/audius-protocol/blob/89e0db2520df13187b809488c4b3aa723a2a7361/.circleci/config.yml#L1017

Reply:
I think the context needs to be at the workflow level instead of the job level